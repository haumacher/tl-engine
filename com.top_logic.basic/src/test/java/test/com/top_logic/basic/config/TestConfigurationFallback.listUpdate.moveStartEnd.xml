<?xml version="1.0" encoding="utf-8" ?>

<fallback-test xmlns:config="http://www.top-logic.com/ns/config/6.0">
	<a-list-configs>
		<aConfig config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
			stringValue="first"
		/>
		<aConfig config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
			stringValue="last"
		/>
	</a-list-configs>
</fallback-test>