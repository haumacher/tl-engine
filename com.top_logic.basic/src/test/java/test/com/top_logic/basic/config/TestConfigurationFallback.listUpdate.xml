<?xml version='1.0' encoding='utf-8'?>
<fallback-test xmlns:config="http://www.top-logic.com/ns/config/6.0">
	<a-list-configs>
		<aConfig config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
			intValue="15" stringValue="one"></aConfig>
		<aConfig config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
			stringValue="two"></aConfig>
		<aConfig config:interface="test.com.top_logic.basic.config.TestConfigurationFallback$A"
			stringValue="three" intValue="-1"></aConfig>
	</a-list-configs>
</fallback-test>