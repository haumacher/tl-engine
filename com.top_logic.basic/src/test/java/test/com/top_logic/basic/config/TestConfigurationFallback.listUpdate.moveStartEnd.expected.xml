<?xml version="1.0" encoding="utf-8" ?>

<fallback-test>
	<a-list-configs>
		<aConfig
			intValue="5"
			stringValue="first"
		/>
		<aConfig stringValue="last"/>
	</a-list-configs>
</fallback-test>